### PR TITLE
Add optional certificate and invite code to regisiterNode

### PR DESCRIPTION
**Pull request by roeierez**
This PR adds the option to supply credentials (certificate + key) or invite code to registerNode.
Grenlight is going to enforce one of them by the end of this month.

### CODE REVIEW

File: libs/sdk-core/src/bridge_generated.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@
     clippy::too_many_arguments
 )]
 // AUTO GENERATED FILE, DO NOT EDIT.
-// Generated by `flutter_rust_bridge`@ 1.75.2.
+// Generated by `flutter_rust_bridge`@ 1.75.1.
```
Comment (ok300) [inline libs/sdk-core/src/bridge_generated.rs:12]: Perhaps a typo? AFAIK @erdemyerebasmaz recently updated to `1.75.2`.

**Review by ok300 — COMMENTED** [review]
The CLI doesn't compile, probably due to the missing new args in `register_node`.

**Comment by roeierez** [comment]
@ok300 Cli now compiles. I will follow with another PR to allow passing the certificate/invite code from cli as well.

File: libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt [hunk 2]
```diff
@@ -82,6 +82,16 @@ fun asNetwork(network: String): Network {
     return Network.valueOf(network.uppercase())
 }
 
+fun asGreenlightCredentials(creds: ReadableMap?) : GreenlightCredentials? {
+ if (creds != null && hasNonNullKey(creds, "deviceKey") && hasNonNullKey(creds, "deviceCert")) {
```
Comment (dangeross) [inline libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt:86]: nit: fix indentation

**Review by dangeross — APPROVED** [review]
Looks good. I tested the RN example with no register credentials.

**Review by ok300 — APPROVED** [review]

**Review by dangeross — APPROVED** [review]

**Pull request merged**